testsuite/fmi-cross-check/importFMUs.py

```diff
@@ -160,9 +160,6 @@ def simulateFMU(omsimulator, testFMUDir, resultDir, modelName, fmiType, luaFile)
   if (not os.path.isfile(os.path.join(resultDir,  modelName+"_out.csv"))) and (exitCode == 0):
     exitCode = 1
 
-  # Delete temp files
-  shutil.rmtree(tempDir, ignore_errors=True)
-
```

Comment:
@AnHeuermann Is this required?

Reply:
No, it's just added to make it more easy to create the PR for the Cross-Check repo. BUt nothing what a `find` from shell can't solve.

Reply:
Ah and to make a rerun possible. But I was using `git clean` anyway, just to be super sure that the working directory is clean.

Reply:
git clean will not touch the temp directory though

Reply:
But OMSimulator is (or should be) cleaning its own garbage. So it should be fine I guess.